src/webview/scaffold-form.html

```diff
@@ -63,6 +63,7 @@ <h2 data-text="this.description()"></h2>
                     data-on-input="this.handleInput(event)"
                     data-attr-type="this.field()[1].format === 'password' ? 'password' : 'text'"
                     data-attr-placeholder="this.field()[1].hint ? this.field()[1].hint : ''"
+                    data-attr-required="this.field()[1].min_length > 0 ? 'true' : 'false'"
```

Comment:
```suggestion
                    data-attr-required="this.field()[1].min_length > 0"
```

Reply:
yikes! good catch. Fixing in immediate follow up...